app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/ApplicationController.scala

```diff
@@ -37,9 +39,9 @@ class ApplicationController @Inject() (
   override val authConnector: AuthConnector,
   mcc: MessagesControllerComponents,
   viewStart: start,
-  migrationStart: migration_start,
```

Comment:
👍

Reply:
I'll check if I can completely remove this view